modules/cluster/addons/prometheus-node-exporter.yaml

```
@@ -157,8 +150,8 @@ spec:
           image: "prom/node-exporter:v0.18.1"
           imagePullPolicy: IfNotPresent
           args:
-            - --path.procfs=/host/proc
-            - --path.sysfs=/host/sys
+            - --collector.filesystem.ignored-mount-points=^/(dev|var/lib/docker|proc|sys|var/lib/kubelet)($|/)
```

Really just a nit but unless for some reason the current regex is more efficient than the following, I'd suggest we'd use this instead, since it is a more common "idiom" and a bit easier to parse.
```suggestion
            - --collector.filesystem.ignored-mount-points=^/(dev|var/lib/docker|proc|sys|var/lib/kubelet)?/$
```

---

Oh I guess I was too slow!... :sweat:

---

noted ...

---

I don't think that's what we want here. It doesn't match for example a mount point like /var/lib/kubelet/pods/path/to/volume.

---

It matches the exact same thing it matched before AFAIK. I see what you mean though, but if you're ignoring `/var/lib/kubelet`, which is covered by the regex, wouldn't your example be ignored as well implicitly?

---

The current regular expression lets us ignore a mount point like `/var/lib/kubelet/pods/path/to/volum`, but you regular expression doesn't. It only matches `/var/lib/kubelet` or `/var/lib/kubelet/`.

---

Oh I see the misunderstanding here, this `collector.filesystem.ignored-mount-points` option does not enforce the resulting string to be the full path of the mount, so if there is no string termination it will match anything that could be appended to the path. Got it.